suite-native/module-add-accounts/src/screens/SelectAccountTypeScreen.tsx

```diff
@@ -34,7 +32,7 @@ const bulletsForKeyPath = (keyPath: TxKeyPath) => (
         values={{
             li: chunks =>
                 chunks.map(row => (
-                    <Box flexDirection="row">
+                    <Box key={`bullet-${row}`} flexDirection="row">
```

Comment:
```suggestion
                    <Box key={row} flexDirection="row">
```

Why is the `bullet-` prefix necessary?

Reply:
It is not. The `${}` is important though, since row can be React.Key | undefined | null. I'll remove `bullet-`

Reply:
if the row can be `undefined` it is not a uniqued identified and should not be used as a key IMO.

Reply:
868b50a92f

Reply:
how does this fixes that?

Reply:
my bad. wrong commit:

https://github.com/trezor/trezor-suite/commit/09fa3a9df396d5afe4a666f668c52d02489034f5